subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/TestSuitesIntegrationTest.groovy

```diff
@@ -480,8 +479,7 @@ class TestSuitesIntegrationTest extends AbstractIntegrationSpec {
         succeeds("checkConfiguration")
     }
 
-    // TODO: Test Framework Selection - Revert this to may NOT in Gradle 8
-    def "test framework MAY be changed once options have been used with test suites"() {
+    def "test framework can not be changed once options have been used with test suites"() {
```

Comment:
Should this also be migrated to the new test class?

Reply:
After a closer look, I think this test is redundant now.  Looks like at least one other test in this class: `"test framework can not be changed once options have been used with test suites"` ought to move though, so I'll do that.